lib/identity_cache/query_api.rb

```diff
@@ -122,7 +122,12 @@ def check_association_scope(association_name)
 
       def record_from_coder(coder) #:nodoc:
         if coder
-          klass = coder[:class]
+          klass =
+            if coder[:class].is_a?(String)
+              coder[:class].constantize
+            else
+              coder[:class]
```

Comment:
What would be a good way to mark(or remember) this as deprecated to make sure we remove it in a future update?